src/offset/local/unix.rs

```diff
@@ -32,68 +32,40 @@ thread_local! {
 }
 
 enum Source {
-    LocalTime { mtime: SystemTime, last_checked: SystemTime },
-    // we don't bother storing the contents of the environment variable in this case.
-    // changing the environment while the process is running is generally not reccomended
-    Environment,
+    LocalTime { mtime: SystemTime },
+    Environment { hash: u64 },
 }
 
-impl Default for Source {
-    fn default() -> Source {
-        // use of var_os avoids allocating, which is nice
```

Comment:
I benchmarked using `var_os` vs `var` and the difference was negligible, while the code was messier, hence the removal of this comment and use of `var_os`